--- PyQtChart_gpl-5.11.3/configure.py.old	2018-10-01 09:41:15.000000000 -0400
+++ PyQtChart_gpl-5.11.3/configure.py	2019-07-20 20:30:56.000000000 -0400
@@ -31,6 +31,7 @@
 import os
 import optparse
 import sys
+from sysconfig import _POSIX_BUILD


 ###############################################################################
@@ -592,7 +593,7 @@
         self.module_dir = sysconfig.get_python_lib(plat_specific=1)
         self.debug = hasattr(sys, 'gettotalrefcount')

-        if sys.platform == 'win32':
+        if sys.platform == 'win32' and not _POSIX_BUILD:
             try:
                 # Python v3.3 and later.
                 base_prefix = sys.base_prefix
@@ -1431,7 +1432,7 @@
     PY_MODULE = %s.pyd
     PY_MODULE_SRC = $(DESTDIR_TARGET)

-    LIBS += -L%s
+    LIBS += -L%s -lpython3.7m
 } else {
     PY_MODULE = %s.so
